portfolio/src/main/java/com/google/sps/servlets/BackgroundChange.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns imageURL to change the background of index.html*/
+@WebServlet("/background-change")
+public class BackgroundChange extends HttpServlet {
+  private List<String> images;
+  private int prevIndex = -1;
+  @Override
```

Comment:
Nit: It's nice to have an extra newline between function declarations. Please add these throughout.

Reply:
I have changed that.